vuu-ui/packages/vuu-shell/src/layout-management/LayoutList.css

```diff
@@ -0,0 +1,55 @@
+.vuuLayoutList-header {
+    color: var(--light-text-primary, #15171B);
+    font-weight: 700;
+    letter-spacing: 0.48px;
+    text-transform: uppercase;
+    display: flex;
+    padding: 16px 0px;
+    align-items: center;
+    align-self: stretch;
+    border-bottom: 1px solid rgba(119, 124, 148, 0.10);
+    line-height: 200%;
+}
+
+.vuuLayoutList-groupName {
+    display: flex;
+    padding-top: 24px;
+    align-items: center;
+    align-self: stretch;
+    color: var(--light-text-secondary, #606477);
+    font-weight: 700;
+    letter-spacing: 0.48px;
+    text-transform: uppercase;
```

Comment:
I know all the Figma examples are written in uppercase, but I'm not sure if we'd want to transform user-provided group names.